### DISCUSSION TITLE
what IS a "minimum" Tailwind repo for a single page, to help people report bugs to you?

### DISCUSSION THREAD

**Discussion by jpwynn** [tailwindlabs/tailwindcss#4537] [Help]
I have to wonder if the requirement to include a minimal repo demonstrating a bug is eliminating a lot of otherwise useful bug reports?

The days of a submitting a simple html file with a few lines that illustrate a css bug are long gone. And in my case, it's a rails app, so not exactly "minimal".

_Might you consider providing somewhere a "sample minimal repo" that people can clone, get running, add their test case, then submit to you as a bug report?_

As a relatively new Tailwinds user, but pretty experienced Bootstrap and Rails developer, I wouldn't even know where to start figuring out what is a minimum Tailwind install to show a single page showing a Tailwinds bug.

**Comment by deadcoder0904** [comment]
If it's a Tailwind bug, there's no question of Rails. It's equivalent to HTML & CSS. So you can create minimal repro on → https://play.tailwindcss.com/
